.github/workflows/create-pr-to-update-release-version.yml

```diff
@@ -0,0 +1,108 @@
+name: PR to update homebrew and minikube
+
+# This action will run after the "Publish container" is succesfully completed
+# https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run
+on:
+  workflow_run:
+    workflows: ["Publish container"]
+    types:
+      - completed
+env:
+  LATEST_HEADLAMP_TAG: latest
+jobs:
+  create_pr_to_upgrade_homebrew_minikube:
+    name: Create PR to upgrade homebrew and minikube
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout headlamp repo 
+        uses: actions/checkout@v2 
+        with:
+          path: headlamp
+          fetch-depth: 0
+      - name: Configure Git 
+        run: | 
+          git config --global user.name 'yolossn'
+          git config --global user.email 'sannagaraj@microsoft.com'
```

Comment:
Not sure if we can infer the user from the action triggering. This way, whoever releases the version will be the one who's getting the user as the PR author. Not super important but just a thought for helping with trailing the actual requests.

Reply:
Only the username is available from the github context, but this step requires both username and email address.
[Refer](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context)

Reply:
Looks like a no-reply GH email can be easily retrieved using the GH rest API.
My insistence on this is that if I create a token for `KINVOLK_REPOS_TOKEN`, I think it gets associated with me. So either we use my own info or we go with the dynamically filled user name + email.

I know you've replied to why we'd need to use a separate token (permissions), but apparently there's a way we can tweak the GITHUB_TOKEN's permissions, e.g.:
https://dev.to/github/the-githubtoken-in-github-actions-how-it-works-change-permissions-customizations-3cgp

Have you looked into this? If you have, let me know, otherwise I think it's worth pursuing, so we can have the user's associated token being used instead of one associated with another user.

Reply:
I checked and couldn't find a way to tweak the GITHUB_TOKEN to access other repos in the organisation, as the token is scoped to this particular repository. For configuring git to use the initiator the GitHub API returns the default email in my case it returns my personal email id. Will it be okay to signoff the commit using personal email ?